File: app/src/main/java/com/example/rodrigo/desafiomobile/gamesList/GamesListView.java
package com.example.rodrigo.desafiomobile.gamesList;

import com.example.rodrigo.desafiomobile.entity.GamesEntity;

import java.util.List;

public interface GamesListView {
    void updateList(List<GamesEntity> gamesLista);

    void AddListRecycler();

    void showMessage(String msg);
    void showLoading();
    void hideLoading();
}

File: app/src/main/java/com/example/rodrigo/desafiomobile/gamesList/GamesListView.kt
package com.example.rodrigo.desafiomobile.gamesList

import com.example.rodrigo.desafiomobile.model.GamesListEntity

interface GamesListView {

    fun displayGames(gamesListEntity: GamesListEntity)
    fun showMessage(msg : String)
    fun showLoading()
    fun hideLoading()
}